### PR TITLE
added pterodactyl

### PR DESCRIPTION
**Pull request by E-4forlife**

### CODE REVIEW

**Review by timothystewart6 — COMMENTED** [review]
hey thanks!  Any way you can change the domain to an example domain?

File: reference_files/traefik-portainer-ssl/traefik/config.yml [hunk 1]
```diff
@@ -101,7 +101,15 @@ http:
         - https-redirectscheme
       tls: {}
       service: opnsense
-   
+    pterodactyl:
+      entryPoints:
+        - "https"
+      rule: "Host(`pterodactyl.local.kmcyber.net`)"
```
Comment (timothystewart6) [inline reference_files/traefik-portainer-ssl/traefik/config.yml:107]: maybe `pterodactyl.example.com` ?

File: reference_files/traefik-portainer-ssl/traefik/config.yml [hunk 2]
```diff
@@ -101,7 +101,15 @@ http:
         - https-redirectscheme
       tls: {}
       service: opnsense
-   
+    pterodactyl:
+      entryPoints:
+        - "https"
+      rule: "Host(`pterodactyl.local.example.com`)"
```
Comment (E-4forlife) [inline reference_files/traefik-portainer-ssl/traefik/config.yml:107]: thanks fixed to the slandered

File: reference_files/traefik-portainer-ssl/traefik/config.yml (hunk 1 above)
Comment (E-4forlife) [reply reference_files/traefik-portainer-ssl/traefik/config.yml]: fixed

**Review by timothystewart6 — APPROVED** [review]
Thank you!

**Comment by timothystewart6** [comment]
Sorry I overlooked this!  Thank you!

**Pull request merged**